### DISCUSSION TITLE
Both low and high pitch adjust for dt and ht

### DISCUSSION THREAD

**Discussion by LosPedros** [ppy/osu#26246] [Ideas]
The problem I have with current pitch adjust is that dt only allows higher pitch and ht only lower.
Would like to be able to select both low and high pitches no matter what mod I have selected

**Comment by cihe13375** [comment]
I think currently the pitch shift from rate-changing mods is determined by physics, i.e. higher speed -> the waveform is "compressed" in time -> higher sound frequency -> higher pitch (for regular DT/HT extra transforms are applied to "cancel" the pitch shift effect).
